src/gourmand/convert.py

```diff
@@ -827,7 +806,7 @@ def integerp(num, approx=0.01):
 
 FRACTION_MATCHER = re.compile(NUM_AND_FRACTION_REGEXP, re.UNICODE)
 
-NUMBER_FINDER_REGEXP = r"(%(NUM_AND_FRACTION_REGEXP)s|%(NUMBER_NO_RANGE_REGEXP)s)(?=($| |[\s]))" % locals()
+NUMBER_FINDER_REGEXP = r"(%(NUM_AND_FRACTION_REGEXP)s|%(NUMBER_NO_RANGE_REGEXP)s)(?=($| |[\s]|-))" % locals()
```

Comment:
Wouldn't we have to escape the `-`? Or is it not recognized as a special pattern character here?